Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -120,3 +123,12 @@
 _hgcal_RawToDigi += hgcalDigis
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toReplaceWith(RawToDigi,_hgcal_RawToDigi)
+
+# RPC New Readout Validation
+from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
```

Comment:
If I read correctly slide 2 in https://indico.cern.ch/event/761884/contributions/3161560/attachments/1729301/2794226/RPC_Digi_Merger_-_RECO_Metting_-_052F102F2018_2.pdf, the "new" hardware layout is only available since 2017, while in 2016 there still was the "legacy" one. If so, I would expect that the ```stage2L1Trigger_2017``` modifier should be used instead.

On the other hand, even with the ```stage2L1Trigger``` modifier the 2016 workflows in the matrix tested here complete apparently correctly. (The twinmux was anyhow already present in 2017).

Could you please check?

Reply:
In my local tests, the size and shape of the new collection for 2016 workflows was too different and the multiplicities were smaller. For 2017 and 2018 it was ok. Maybe there is something missing in 2016.

For our monitoring proposal, ```stage2L1Trigger_2017``` seems safer. Local tests were good.